neard/tests/node_cluster.rs

```diff
@@ -0,0 +1,106 @@
+use futures::future;
+
+use near_actix_test_utils::{run_actix_until_stop, spawn_interruptible as spawn};
```

Comment:
Let's not hide the fact that we use `spawn_interruptible` helper. ("Explicit is better than implicit", by the Zen of Python)

```suggestion
use near_actix_test_utils::{run_actix_until_stop, spawn_interruptible};
```